guides/0113-random_source.md

```diff
@@ -0,0 +1,69 @@
+# random_source
+
+* type: producer
+* output: either integer, real or boolean messages
+
+Producer block that generates random messages of a specific type with a fixed key.
+
+# Properties
+
+* `key`: A unicode string that will be used as key in the generated messages. (required, string)
+* `type` The type of data that will be generated. Supported types are `integer`, `real` and `boolean`. (required, string)
+* `delay_ms`: If present, the block will wait `delay_ms` before emitting a new sample. (optional, integer)
```

Comment:
Let's check if we have a default value for this one

Reply:
Default is `nil` or key not configured. If not configured there will be no delay between messages